backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/process/ProcessInfo.java

```diff
@@ -248,6 +249,8 @@ private ProcessInfo(
 	@Getter
 	private final boolean reportingProcess;
 	@Getter
+	private final boolean doNotQuoteRows;
```

Comment:
why not having this flag into de.metas.process.SpreadsheetExportOptions (see spreadsheetExportOptions field)?